index.html

```diff
@@ -327,13 +328,48 @@ <h2>Get in Touch</h2>
             <a href="mailto:rebaone02khanyi@gmail.com" class="btn">Contact Me</a>
         </div>  
 
-    <!--My socials-->
+        <!--My socials-->
+    
     <div class="social-links">
         <h3>Socials</h3>
         <a href="https://github.com/KhanyiCodes" target="_blank"><img src="images/logos/github.png" alt="GitHub Logo"></a>
         <a href="https://www.linkedin.com/in/rebaone-khanyisile-cynthia-vilakazi-37407528a/" target="_blank"><img src="images/logos/linkedin.png" alt="LinkedIn Logo"></a>
     </div>
+    
     </section>
+    <!--Chatbot-->
+    <section>
+    <div class="chat-container">
+        <div class="chat-header">
+            <div class="chat-header title">Chatbot</div>
+            <div class="chat-header close-btn" onclick="toggleChat()">X</div>
+        </div>
+        <div class="chat-body" id="chat-body">
+            <div id="chat-messages">
+                <div class="bot-message">
+                    Hello! Welcome to Rebaone's portfolio. What can I help you find? 
+                </div>
+                <div class="bot-message">
+                    <button class="option-button" onclick="sendMessage('Home')">Home</button>
+                    <button class="option-button" onclick="sendMessage('About')">About Me</button>
+                    <button class="option-button" onclick="sendMessage('Skills')">Skills</button>
+                    <button class="option-button" onclick="sendMessage('Contact')">Contact Me</button>
+                </div>
+            </div>

```

Comment:
@bokangkhumalo Do you think she should change this code?

Reply:
Yes, must NOT include the nav bar in the chatbot